pages/[...slug].tsx

```diff
@@ -255,7 +255,7 @@ function renderSidebarItems(
  * Generate `OnThisPage` items based on sub-headings within document
  *
  */
-function renderOnThisPageItems(
+function useRenderOnThisPageItems(
```

Comment:
Hmm it's not really being used as a hook though – can it be turned into a component?

Reply:
Turned it into a functional component.